packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -155,7 +162,9 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
             },
         },
         paper: {
-            boxShadow: shadows[1],
+            boxShadow: "0px 0px 40px 0px #0000001A",
```

Comment:
Why isn't it possible to use one of the theme's shadows. @jamesricky should we change that?

Reply:
See https://github.com/vivid-planet/comet/pull/2457#discussion_r1751351862